test/autotest.py

```diff
@@ -525,7 +527,9 @@ def testRestart():
       if i.endswith(".dmtcp"):
         cmd+= " "+ckptDir+"/"+i
     #run restart and test if it worked
-    procs.append(launch(cmd))
+    procs.append(runCmd(cmd))
+    #wait and see if process will die shortly after restart
+    sleep(S*SLOW)
```

Comment:
Agreed.  Note that in the previous line note above, I moved the `sleep()' to after the `WAITFOR()`.  I agree that we should do the same here.  Thanks for the correction.